### ISSUE TITLE
Abnormal Interest

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7274]
**Preferred term label:**
Abnormal Interest

**Synonyms**

**Definition (free text, please give PubMed ID)**
@pnrobinson, how do I fill this out if I mostly intend this as a parent term for other behavioral phenotypes that are more specifically defined in the literature? Is it fair to just use an intuitive definition for a term like this?

**Parent term (use hpo.jax.org/app)**
Behavioral Abnormality (HP:0000708)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
I would try to find some description about this category. A quick search finds e.g., this https://pubmed.ncbi.nlm.nih.gov/18427535/, that mentions some examples of "pathological interests", but it would be better to have a definition that denotes what exactly "abnormal" means in this context. Perhaps the chapter intro in DSM-V? This is something that we can address at the workshop, and we could add the term with a bare-boines definition withthe intention of adding a better definition later on. @bcoleman96

**Comment by bcoleman96** [comment]
@pnrobinson , I was having trouble finding a definition because (1) most publications with terms like this do not give a definition and (2) descriptions of the phenotype are usually specific to the disease of interest. In the case of the above article, the definition relates to schizophrenia but we will want a broader definition that can include certain findings in autism and OCD. As you said, I think it makes sense to ask the clinical experts. I also wonder if we can construct a better definition once we've been more thorough about considering what child terms would be under it. Though maybe this is a backward way to do things.

**Comment by pnrobinson** [comment]
added term with definition:
```
Any deviation from normal intensity or subject of a person's interests, 
which can be defined as a state of giving selective attention to something.
```
